doc/api/next_api_changes/development/00001-EFS.rst

```diff
@@ -0,0 +1,6 @@
+- The signature and implementation of the `set_hover()` and `get_hover()` methods of the `Artist` class, as well as the `HoverEvent(Event)` Class, have been added:
+
+  .. versionchanged:: 3.7.2
```

Comment:
versionchanged directive does not make sense in an API note (everything here is a version changed)

Additionally since it is an API change, it will be going in to the next minor version if/when it is merged (currently 3.8.0) rather than a patch release like 3.7.2